ucm2/Allwinner/A64/PinePhone/VoiceCall.conf

```diff
@@ -0,0 +1,149 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='Headphone Playback Switch' off"
+		cset "name='Headphone Source Playback Route' DAC"
+		cset "name='Line In Playback Switch' off"
+		cset "name='Line Out Playback Switch' off"
+		cset "name='Line Out Source Playback Route' Mono Differential"
+		cset "name='Mic1 Playback Switch' off"
+		cset "name='Mic2 Playback Switch' off"
+		cset "name='AIF1 DA0 Playback Volume' 160"
+		cset "name='AIF2 DAC Playback Volume' 160"
+		cset "name='AIF3 ADC Source Capture Route' None"
+		cset "name='AIF2 DAC Source Playback Route' None"
```

Comment:
`None` is invalid, per https://github.com/alsa-project/alsa-ucm-conf/pull/134/files#r802213404.

Reply:
Introduced in https://gitlab.com/postmarketOS/pmaports/-/merge_requests/1614, and looks to pre-date smaeul's work.